tldw_Server_API/tests/AuthNZ_SQLite/test_org_stt_settings_repo_sqlite.py

```
@@ -0,0 +1,96 @@
+    reset_settings()
+    await reset_db_pool()
+
+    pool = await get_db_pool()
+    ensure_authnz_tables(Path(str(db_path)))
+
+    users_db = UsersDB(pool)
+    await users_db.initialize()
+    created_user = await users_db.create_user(
+        username="org-stt-sqlite",
+        email="org-stt-sqlite@example.com",
+        password_hash="hashed-password",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password_hash"

(S106)